TradeItIosTicketSDK2/TradeItOrdersViewController.swift

```diff
@@ -121,7 +119,19 @@ class TradeItOrdersViewController: TradeItViewController, TradeItOrdersTableDele
             onCancelActionTapped: {}
         )
     }
-    
+
+    private var cancelOrderSecurityHandler: (() -> Void)?
+    internal func handleCancelOrderSecurityResponse() {
+        if let cancelOrderSecurityHandler = self.cancelOrderSecurityHandler {
+            cancelOrderSecurityHandler()
+        } else {
+            return self.alertManager.showError(
+                TradeItErrorResult.error(withSystemMessage: "Canceling order could not be submitted please try again."),
```

Comment:
This will show the default error message to the user. `systemMessage` is logged to console. Re-word to something like: `Your cancel order request could not be submitted. Please try again.`